doc/cli/model.rst

```diff
@@ -77,6 +79,18 @@ By default, YOGA optimize the 3D models and its textures. The optimizations can
     yoga  model  --no-graph-optimization  --no-meshes-optimization  --no-textures-optimization  input.fbx  output.glb
 
 
+Disabling Postprocesses
+-----------------------
+
+By default, YOGA use several Postprocesses, they can be disabled using the following options:
```

Comment:
By default, YOGA use several postprocesses, some can be disabled using the following options: